http-generator-helidon/src/main/java/io/avaje/http/generator/helidon/nima/ControllerWriter.java

```diff
@@ -108,9 +111,6 @@ private void writeRoutes(List<ControllerMethodWriter> methods) {
 
     for (final ControllerMethodWriter methodWriter : methods) {
       methodWriter.writeRule();
-      if (!reader.isDocHidden()) {
```

Comment:
the problem was that it was setting the openApi information before the parameter  was determined to be a path